tutorials/clinical/30_ecog.py

```diff
@@ -134,6 +135,31 @@
     envelope=True).resample(sfreq)
 gamma_info = gamma_power_t.info
 
+# %%
+# Plot Gamma Power on cortical sensors
+# --------------------------------------
+#
+# We will now use evoked gamma power to plot on the cortical surface.
+# Therefore we extract the evoked time sample at 15s and normalize
+# it in a range of 0 to 1 in order to map it using a matplotlib colormap.
+
+gamma_power_vec = np.array(gamma_power_t.to_data_frame().iloc[20, 1:])
```

Comment:
using `iloc` here is a bit opaque / magic, users won't know why you picked those indices without inspecting the dataframe (which isn't currently created/saved as a separate variable).  Here's a suggestion for something more straightforward / self-documenting:

```suggestion
gamma_power_vec = gamma_power_t.to_data_frame(index='time').loc[15]
```